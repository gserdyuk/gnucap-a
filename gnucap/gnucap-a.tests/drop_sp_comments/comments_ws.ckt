* simlpe multiline element circuit
.options drop_spice_comments=yes

r1 1 2 50k
r2 1 
* first line of comments
+ 2 50k
r3 1
* comment line 1
* comment line 2
+ 2 50k

r4 1
* comment line 1
* comment line 2 and empty line then

+ 2 50k

r5 
* comment angain

* comment after blank line
+1 2 50K

r6
* comment
      
* comment after blank line with whitespaces
+ 1 2 50k

r7
      
+ 1 2 50k

r8
* 1st comment line
     
*second comment line after whitespaces
      
+ 1 2 50k

.list
.dc
.end

