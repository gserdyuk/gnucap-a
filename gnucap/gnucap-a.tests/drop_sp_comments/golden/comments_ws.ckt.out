Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-04-09
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* simlpe multiline element circuit 
r1 ( 1 2 )  50.K
r2 ( 1 2 )  50.K
r3 ( 1 2 )  50.K
r4 ( 1 2 )  50.K
r5 ( 1 2 )  50.K
r6 ( 1 2 )  50.K
r7 ( 1 2 )  50.K
r8 ( 1 2 )  50.K
#          
 0.        
